### ISSUE TITLE
Question about Language to object cls loss(lang_loss)

### ISSUE THREAD

**Issue by cheng052** [daveredrum/ScanRefer#12]
Hi, zhenyu,

Thanks for your good work. However, I got a question when training the network.

<img width="624" alt="lang-loss" src="https://user-images.githubusercontent.com/34396754/107459790-df8e3180-6b91-11eb-980e-de422abdba60.png">
<img width="617" alt="lang-acc" src="https://user-images.githubusercontent.com/34396754/107460030-588d8900-6b92-11eb-87b1-59d17520c6d5.png">

Firstly, I train a model from scratch with the command `python script/train.py --use_multiview --use_normal`, which enable the language to object classification loss(lang_loss).

The lang_loss and lang_acc are shown in the pictures. 

It is very strange that **the language loss on validation set is much lower than the loss on train set**, which violates the basic principals of machine learning.

Also, **the lang_acc is much higher on validation set than the acc on train set**, which makes me more confused.

Can you check this problem? It can be of great help.

**Comment by daveredrum** [comment]
Hi @cheng052,

Thanks for your interest in our work! We apply a dropout (p=0.5) in the language classifier to prevent overfitting, as the model will easily overfit on the language classification subtask. That's why training loss stops to drop at some point and validation loss is way lower than the training. We've tested the model without setting the dropout, but it turned out to be a huge failure.

**Comment by cheng052** [comment]
Hi @daveredrum,

Thanks for your prompt reply. It does solve my problem, thanks!
